helpers/localize.js

```diff
@@ -0,0 +1,28 @@
+import { html } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+export const allowedTags = Object.freeze(['d2l-link', 'd2l-tooltip-help', 'p', 'br', 'b', 'strong', 'i', 'em']);
+
+const markupError = `localizeHTML() rich-text replacements must use markup templates with only the following allowed elements: ${allowedTags}. [link to docs]`;
+const disallowedTagsRegex = new RegExp(`<(?!/?(${allowedTags.join('|')})?[\\s>])`);
```

Comment:
```suggestion
const validTerminators = '([>\\s/]|$)';
const disallowedTagsRegex = new RegExp(`<(?!/?(${allowedTags.join('|')})?${validTerminators})`);
```

Reply:
Kind of hurts my head more to break these out and try to piece them back together. Also, `allowedAfterTriangleBracket` is allowed in the code, but _not_ allowed in order to match the RegEx 🤯.

(`allowedTags` isn't a piece of the RegEx, and in fact is manipulated inline to be useful to it)

Reply:
I'm not attached to it, though I think it improves the readability. `allowedAfterTriangleBracket` is what's allowed to be there - `disallowedTagsRegex` negates that, as the name implies. Regex can be hard to read and even harder to document, so I like to self-document with variable names.

Anyway, I think separating the `validTerminators` at least is beneficial.